test-cases/tests/alerts/c13-verify-fuse-alert.md

```diff
@@ -0,0 +1,23 @@
+---
+targets:
+  - 2.6.0
+---
+
+# C13 - Verify Fuse alert
+
+## Description
+
+More info: <https://issues.redhat.com/browse/INTLY-9047>
+
+## Steps
+
+1. Scale down `syndesis-db` for Fuse Online
+   1. Login as kubeadmin
+   2. **Workloads > Deployment Configs**
+   3. Select `redhat-rhmi-fuse` project
+   4. Scale `syndesis-db` down to 0 pods
```

Comment:
@jhellar syndesis-db won't exist in RHMI 2.x as RDS is used.
This looks like a 1.x only issue (i've updated the jira fixVersion).